google/api/annotations.proto

```diff
@@ -67,23 +79,18 @@ extend google.protobuf.FieldOptions {
   // The fully qualified message name of the type that this references.
```

Comment:
Sorry! just seen one more thing...
Why does this need to be fully qualified? I would think we can allow name-only (ie not fully-qualified) if the message/resource-definition is in the same namespace. This would then match the specification of `resource_references` in ResourceSet

Reply:
It does not need to be fully qualified. I will update that comment!